website/docs/r/compute_fleet.html.markdown

```diff
@@ -0,0 +1,573 @@
+
+---
+
+A `os_profile` block supports the following:
+
+* `custom_data_base64` - (Optional) The base64-encoded Custom Data which should be used for the Compute Fleet. Changing this forces a new resource to be created.
+
+* `linux_configuration` - (Optional) A `linux_configuration` block as defined above. Changing this forces a new resource to be created.
+
+* `windows_configuration` - (Optional) A `windows_configuration` block as defined below. Changing this forces a new resource to be created.
+
+---
+
+A `additional_capabilities` block supports the following:
+
+* `hibernation_enabled` - (Optional) Whether to enable the hibernation capability on the Compute Fleet.  Defaults to `false`. Changing this forces a new resource to be created.
+
+* `ultra_ssd_enabled` - (Optional) Whether to enable Data Disks of the `UltraSSD_LRS` storage account type on this Compute Fleet. Defaults to `false`. Changing this forces a new resource to be created.
+
+---
+
+A `additional_unattend_content` block supports the following:
+
+* `xml` - (Required) Specifies the XML formatted content that is added to the unattend.xml file for the specified path and component. Changing this forces a new resource to be created.
+
+* `setting` - (Required) Specifies the name of the setting to which the content applies. Possible values are `AutoLogon` and `FirstLogonCommands`. Changing this forces a new resource to be created.
+
+---
+
+A (Windows) `certificate` block supports the following:
+
+* `url` - (Required) The secret URL of a key vault certificate. Changing this forces a new resource to be created.
+
+* `store` - (Optional) The certificate store on the virtual machine where the certificate should be added. Changing this forces a new resource to be created.
+
+---
+
+A (Linux) `certificate` block supports the following:
+
+* `url` - (Required) The secret URL of a key vault certificate. Changing this forces a new resource to be created.
+
+---
+
+A `data_disk` block supports the following:
+
+* `create_option` - (Required) The create option which should be used for the data disk. Possible values are `Empty` and `FromImage`. Changing this forces a new resource to be created.
+
+-> **Note:** `FromImage` should only be used if the source image includes data disks.
+
+* `caching` - (Optional) The type of caching which should be used for the data disk. Possible values are `ReadOnly` and `ReadWrite`. Changing this forces a new resource to be created.
+
+* `delete_option` - (Optional) Specify what happens to the data disk when the virtual machine is deleted. Possible values are `Delete` and `Detach`. Changing this forces a new resource to be created.
+
+* `disk_encryption_set_id` - (Optional) The ID of the disk encryption set which should be used to encrypt the data disk. Changing this forces a new resource to be created.
+
+* `disk_size_in_gib` - (Optional) The size of the data disk which should be created.  Changing this forces a new resource to be created.
```

Comment:
Maybe indicate the unit once more in the description to emphasize the differentiation from GB?
```suggestion
* `disk_size_in_gib` - (Optional) The size of the data disk which should be created in GiB. Changing this forces a new resource to be created.
```